lib/open_food_network/user_balance_calculator.rb

```diff
@@ -6,7 +6,7 @@ def initialize(email, distributor)
     end
 
     def balance
-      -completed_orders.to_a.sum(&:outstanding_balance)
+      -completed_orders.to_a.sum(&:old_outstanding_balance)
```

Comment:
We're not using the new balance here?

Reply:
yes, because this exactly the class we set out to replace in the first place (long ago :sweat: ), which was the root cause of the awful performance